newtmgr/cli/fs.go

```diff
@@ -94,7 +94,7 @@ func fsUploadRunCmd(cmd *cobra.Command, args []string) {
 	c.Name = args[1]
 	c.Data = data
 	c.ProgressCb = func(c *xact.FsUploadCmd, rsp *nmp.FsUploadRsp) {
-		fmt.Printf("%d\n", rsp.Off)
```

Comment:
Why did we change this?

Reply:
newline causes a bunch of semi-useless numbers to scroll and scroll and scroll.  CR keeps the output on a single line.
(The upload can take a while ... so its good to have *some* sign of life while the upload is going, but too much output is not either)